README.md

```diff
@@ -75,71 +75,60 @@ If the application says your device does not support biometrics yet, you have to
 Open the settings and add a PIN as well as a fingerprint as shown below (PIN is required for fingerprint).
 ![image](https://user-images.githubusercontent.com/23581140/232045115-86943a1a-c00a-48c3-bdc8-3f98daa962bc.png)
 
-
 ## 4. iOS
 
-### 4.1. Configuring iOS Associated Domains
+### 4.1. Add iOS app to developer panel
+
+Inside the developer panel, go to [Settings -> Credentials -> Native Apps](https://app.corbado.com/app/settings/credentials/native-apps) and click on 'Add new'. There you need to enter the bundle identifier (this sample application uses `com.corbado.demo` as default bundle identifier) as well as the app identifier prefix of your iOS Application. This can be obtained by going to your [Apple Developer Certificates, Identifier & Profiles](https://developer.apple.com/account/resources/identifiers/bundleId) associated with your Apple Developer account, and finding the corresponding identifier.
```

Comment:
Can we rename the bundle identifier to "com.corbado.passkeys", so that it's the same like on Android?